sidebars.js

```diff
@@ -477,35 +477,82 @@ module.exports = {
         "apis-clients/operate-api/index",
         require("./docs/apis-clients/tasklist-api/sidebar-schema"),
         "apis-clients/console-api-reference",
+
         {
           "Optimize API (REST)": [
```

Comment:
These sections now need cross-instance links, instead of document IDs, just like in the optimize_sidebars (but in the opposite direction).